app/frontend/package.json

```diff
@@ -49,6 +49,7 @@
           "@typescript-eslint/no-unused-vars": [
             "warn",
             {
+              "argsIgnorePattern": "theme",
```

Comment:
I think since we are ok with having `useStyles` skeletons in our components to make it easier to make "good first issues", this is related so I think it's ok to add to the exception too, as most styling will probably need things from the theme once they **are** being applied.